lib/webmachine/cookie.rb

```diff
@@ -103,9 +103,7 @@ def to_s
     private
 
     def rfc2822(time)
-      wday = Time::RFC2822_DAY_NAME[time.wday]
-      mon = Time::RFC2822_MONTH_NAME[time.mon - 1]
-      time.strftime("#{wday}, %d-#{mon}-%Y %H:%M:%S GMT")
+      time.strftime('%a, %d-%b-%Y %T GMT')
```

Comment:
The `Time::RFC2822_DAY_NAME` constant is not available in Ruby 3.1.

Replace it with a more-straightforward `strftime` implementation.

Reply:
@orien 
In practice it probably does not matter, but I think this implementation might not be 100% correct (both now, and before). The format for the date (`%d-#{mon}-%Y`)  is using `-` instead of a space(` `) to separate the components of the date. From what I saw, the RFC suggests a space should be used, [so does MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expires). I'm fine to merge it as-is, but maybe this is worth considering. Please let me know what you think, and thanks!

Reply:
Relevant RFC: https://www.rfc-editor.org/rfc/rfc2616#section-3.3.1
It does look like space should be used, and that `-` can be used but with a date that's formatted differently.

Reply:
This is a great observation. I think it _should_ simplify the implementation. I'll investigate…

Reply:
Updated in [`10851c5`](https://github.com/webmachine/webmachine-ruby/pull/263/commits/10851c5ccaee05bbccef66fa11ec4f1567816d9b).